pkg/pdatatest/ptracetest/traces_test.go

```diff
@@ -75,7 +203,7 @@ func TestCompareTraces(t *testing.T) {
 			if tc.withoutOptions == nil {
 				assert.NoError(t, err)
 			} else {
-				assert.EqualError(t, tc.withoutOptions, err.Error())
+				assert.EqualError(t, err, tc.withoutOptions.Error())
```

Comment:
Thank's for fixing it. I thought the first argument was expected value, appeared to be it's the second one. Feel free to fix other tests if you have a chance

Reply:
Actually, nevermind. I'll fix in a PR that I'm currently working on